src/transport.ts

```diff
@@ -19,53 +19,93 @@ import * as sdp from './sdp.js'
 import { WebRTCStream } from './stream.js'
 
 const log = logger('libp2p:webrtc:transport')
+
+/**
+ * The time to wait, in milliseconds, for the data channel handshake to complete
+ */
 const HANDSHAKE_TIMEOUT_MS = 10000
-const WEBRTC_CODE: number = 280
-const CERTHASH_CODE: number = 466
 
+/**
+ * Created by converting the hexadecimal protocol code to an integer.
```

Comment:
but weird they do not just write it as hex literals, but ok, seems you're just following the docs here to the point